cicd/ingest.csv

```diff
@@ -171,8 +171,8 @@ city=Boston | stats count AS Count BY http_method | eval newField=lower(http_met
 "* | head 2 | stats count",now-1d,now,*,countRecord:count(*):*,eq,2,Splunk QL
 "* | head | stats count",now-1d,now,*,countRecord:count(*):*,eq,10,Splunk QL
 "city=Boston | head limit=3 | stats count",now-1d,now,*,countRecord:count(*):*,eq,3,Splunk QL
-"app_name = Bracecould | head gender=""female"" | stats count",now-1d,now,*,countRecord:count(*):*,eq,3,Splunk QL
-"app_name = Bracecould | head gender=""female"" keeplast=true | stats count",now-1d,now,*,countRecord:count(*):*,eq,4,Splunk QL
+"app_name = Bracecould | sort gender | head gender=""female"" | stats count",now-1d,now,*,countRecord:count(*):*,eq,3,Splunk QL
+"app_name = Bracecould | sort gender | head gender=""female"" keeplast=true | stats count",now-1d,now,*,countRecord:count(*):*,eq,4,Splunk QL
```

Comment:
Why did you add sorting? Since this PR is about fixing an issue, you probably shouldn't change any of the tests unless there's a good reason to. But it looks like the query is supposed to give the same results either way, right?

Reply:
So, when multiple segments are used, the order of records is not certain. This test picks the records if gender is female. In the multiple segment case, the gender male often comes up as first record and this query would return zero. To have a consistent query for checking the results we would need a sort. In the default case when a single segment ingest is done. records where female is the gender are ingested first.